src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -963,11 +963,12 @@ private InstructorFeedbackResultsQuestionTable buildQuestionTableAndResponseRows
             }
             
             // If question specific sorting is not needed, responses are sorted
-            // by default order (first by team name, then by display name)
+            // in the following order (first by recipient name, then by team name,
+            // then by display name)
             if (questionDetails.isQuestionSpecificSortingRequired()) {
                 Collections.sort(responseRows, questionDetails.getResponseRowsSortOrder());
             } else {
-                responseRows = InstructorFeedbackResultsResponseRow.sortListWithDefaultOrder(responseRows);
+                responseRows = InstructorFeedbackResultsResponseRow.sortListByRecipientNameThenDefaultOrder(responseRows);
```

Comment:
@kanghj Wait, so we are not sorting the gray table according to recipient, but sorting the white table(the one above with the question mark) according to alphabetical order? Meaning if I refer to your table, instead of BBBB, CCCC, AAAA it should be AAAA, BBBB, CCCC?